src/scripts/elementFactory.js

```diff
@@ -1,10 +1,11 @@
 // function to create any elements (el) with any content (cont or null) and you can designate if it has a child
 const elementCreator = {
-  elementFactory(el, cont, clazz, id, ...children) {
+  elementFactory(el, cont, clazz, id, link, ...children) {

```

Comment:
New parameter  to the element factory function. Updates will be needed for news and to do